wazo_router_confd/auth.py

```diff
@@ -0,0 +1,116 @@
+from typing import Optional, List
+
+from aiohttp import ClientSession
+from aiohttp import TCPConnector  # type: ignore
+
+from fastapi import FastAPI
+from starlette.requests import Request
+from starlette.responses import Response
+
+
+X_AUTH_TOKEN_HEADER = 'X-Auth-Token'
+WAZO_TENANT = 'Wazo-Tenant'
+
+
+class Principal(object):
```

Comment:
A very good candidate for https://docs.python.org/3/library/dataclasses.html

No more need to define __init__ and __repr__.

Reply:
Cool stuff, didn't know it exist :)